### ISSUE TITLE
VA Forms System Core [Component]: Step 10 - ID attribute values must be unique

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va-forms-system-core#547]
### Point of contact

David Kennedy (DK)

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

There are a few spots in the form where ID attributes aren't unique. These are:

```html
<section id="claimant-information">
<section id="veteran-information">
<section id="veteran-information-burial">
<section id="military-history-service-periods">
<section id="military-history-previous-names">
<section id="benefits-selection">
<section id="benefits-burial-allowance">
<section id="benefits-plot-allowance">
```

It looks like each of these has a h2 with the same ID. I'm not sure if this is an individual demo form problem with the implementation or something the form does on its own.

### Reproduction steps

1. View review page.
2. Search for the IDs above in the HTML, and note the multiple instances.

### Proposed solution or next steps

Only refer to an ID once.

### References, articles, or WCAG support

WCAG 4.1.1 Parsing

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [X] Axe-core
- [ ] Something else

**Comment by obliviga** [comment]
Hi @davidakennedy, this issue should be addressed in my PR here: https://github.com/department-of-veterans-affairs/va-forms-system-core/pull/565

I added another feature in my PR that ensures focus is placed back on the Edit button when "Back to Review Page" is activated from the Edit View. Would this behavior improve accessibility?

**Comment by davidakennedy** [comment]
> I added another feature in my PR that ensures focus is placed back on the Edit button when "Back to Review Page" is activated from the Edit View. Would this behavior improve accessibility?

Hi @obliviga, thanks for the work on this! Yes, I think that's a good call. It's a nicety/good UX to put them back where they started. Good call!

**Comment by obliviga** [comment]
Verified fix has been deployed to the main branch.
